server/gitrest/packages/gitrest/config.json

```diff
@@ -20,6 +20,7 @@
             "name": "nodegit"
         },
         "persistLatestFullSummary": false,
+        "enableLowIoWrite": "initial",
```

Comment:
Unfortunately, we currently have no UTs for summaries routes in Gitrest :(

Reply:
I added it to the values.yml and configmap, though. I also added `enableRepositoryManagerMetrics` to the config.json since it was missing there, but existed in the k8s charts.

Reply:
Thanks! I probably missed `enableRepositoryManagerMetrics` in the config.json in my PR earlier 😅